compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/partition_cut.rs

```diff
@@ -71,22 +157,54 @@ impl PartitionCut {
                 }
                 Some(PartitionIndex(self.p_cut.len()))
             }
+            Operator::ChangePartition {
```

Comment:
```
ChangePartition(Some(partition), None)
| ChangePartition(None, Some(partition))
```

Reply:
and maybe we should reject non supported case to detect some errors from CP/compiler